frontend/javascripts/oxalis/controller/mesh_controller.ts

```diff
@@ -0,0 +1,167 @@
+// @ts-expect-error ts-migrate(7016) FIXME: Could not find a declaration file for module 'twee... Remove this comment to see the full error message
+import TWEEN from "tween.js";
+import * as THREE from "three";
+import app from "app";
+import { mergeVertices } from "libs/BufferGeometryUtils";
+import _ from "lodash";
+import type { Vector3 } from "oxalis/constants";
+import CustomLOD from "oxalis/controller/custom_lod";
+import { getSegmentColorAsHSLA } from "oxalis/model/accessors/volumetracing_accessor";
+import { NO_LOD_MESH_INDEX } from "oxalis/model/sagas/isosurface_saga";
+import Store from "oxalis/store";
+
+export default class MeshController {
```

Comment:
I would rename this into "IsosurfaceController" or "IsosurfaceMeshController". Because there are still other meshes in the `SceneController` like the bounding boxes and more (see `SceneController.createMeshes`) that are not handled by this class.

Reply:
good point! I renamed it to `SegmentMeshController`, since I feel like we should get rid of the term "isosurface" in the long run (the UI already uses meshes mostly).